spec/model/topic_spec.rb

```diff
@@ -35,9 +56,9 @@
     )
   end
 
-  describe '.open_alerts' do
+  describe '.open_alerts_count' do
     it 'should return the right count' do
-      expect(Topic.open_alerts).to contain_exactly(firing_alert.topic)
+      expect(Topic.open_alerts_count).to eq(2)
     end
```

Comment:
I guess this is the

> fixes a bug where the previous open alerts count did not account for multiple alerts in a given topic leading to an inflated count in certain cases

? Any chance to modify the test description to describe it rather than just "return the right count" since "right" would be ambiguous to the reader without context / gitblame?

Reply:
Do you mean I should include the correct count in the description?

Reply:
I've updated the description to reflect what we're counting in this case.